test/e2e/framework/test_context.go

```diff
@@ -286,7 +289,8 @@ func RegisterCommonFlags(flags *flag.FlagSet) {
 	flags.BoolVar(&TestContext.DeleteNamespaceOnFailure, "delete-namespace-on-failure", true, "If true, framework will delete test namespace on failure. Used only during test debugging.")
 	flags.IntVar(&TestContext.AllowedNotReadyNodes, "allowed-not-ready-nodes", 0, "If non-zero, framework will allow for that many non-ready nodes when checking for all ready nodes.")
 
-	flags.StringVar(&TestContext.Host, "host", "", fmt.Sprintf("The host, or apiserver, to connect to. Will default to %s if this argument and --kubeconfig are not set", defaultHost))
+	flags.StringVar(&TestContext.Host, "host", "", fmt.Sprintf("The host, or apiserver, to connect to. Will default to %s if this argument and --kubeconfig are not set.", defaultHost))
+	flags.StringVar(&TestContext.BearerToken, "bearer-token", defaultBearerToken, "The bearer token to authenticate with.")
```

Comment:
Could we clarify in the description "where" this authn / flag is needed - i.e. node e2e.

Reply:
should we only add this flag when running node e2e tests?

Reply:
i'd say yes.